tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "allowJs": true,
     "sourceMap": true,
     "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "outDir": "./dist",
```

Comment:
Was something not working prior to this?  I'm curious about why this was needed now, but not before.

Reply:
It's not needed, and was removed.